CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 [Unreleased]: https://github.com/atomist/rug/compare/1.0.0-m.3...HEAD
 
+### Changed
+
+-   **BREAKING** Removed `label` from `Presentable` instruction
```

Comment:
We _could_ make this a non-breaking change by just introducing a new Presentable type, `UnlabelledPresentable` and use that instead for the slack stuff.